usecases.md

```diff
@@ -140,6 +140,74 @@ The table below shows the attributes for three different examples. We assume all
 | Site | Sensor X (America/Denver) | Power Plant Y (Etc/GMT+7) | Aggregate Z (America/Phoenix) |
 
 
+### Probabilistic forecasts
+{: .anchor}
+
+A *probabilistic forecast* in the Solar Forecast Arbiter is one or more
+[forecast evaluation time series](#forecastattrs) that describe part or all of a probability
+distribution.
+
+Let us consider a few simple examples. A probabilistic forecast may be the probability that
+generation is less than 10 MW. This probabilistic forecast is described by a
```

Comment:
Actually, this first example is a deterministic forecast, because each forecast data point is a single value, its just forecasting a statistic different than the average.

Reply:
I disagree. The forecast has units of probability and requires a different set of metrics. I think the single vs. multiple values is not the defining feature of a probabilistic forecast. There is also the implied complementary probability forecast to consider.